staging/src/k8s.io/apiserver/plugin/pkg/authenticator/password/passwordfile/passwordfile.go

```diff
@@ -85,7 +86,7 @@ func (a *PasswordAuthenticator) AuthenticatePassword(ctx context.Context, userna
 	if !ok {
 		return nil, false, nil
 	}
-	if user.password != password {
+	if subtle.ConstantTimeCompare([]byte(user.password), []byte(password)) == 0 {
```

Comment:
arghghghgh

Reply:
?

Reply:
translation: *incoherent despair that we could possibly have had such a bug in 2019*

We also don't salt+hash the plain text passwords in the csv file, sigh...